server/src/main/java/org/opensearch/tasks/consumer/SearchShardTaskDetailsLogMessage.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.tasks.consumer;
+
+import org.opensearch.action.search.SearchShardTask;
+import org.opensearch.common.logging.OpenSearchLogMessage;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Search shard task information that will be extracted from Task and converted into
+ * format that will be logged
```

Comment:
Please include the `@opensearch.internal` annotation assuming this is never meant to be extended by anything outside of core.

Reply:
Done